### ISSUE TITLE
nice logger

### ISSUE THREAD

**Issue by Izuyazep** [Runduen/bbystealer#1]

**Comment by Pandaxyzzz** [comment]
No, it's not a "nice logger" it's a LITERALLY pirate stealer renamed from the repository, literally in code it puts information about what a pirate stealer is, not to mention the `README.MD` and credits. If you want to thank the project, thank: Stanley, Bytixo, Autist69420 and HideakiAtsuyo.
